### ISSUE TITLE
Upon Upload of Image - Appropriately Sized Image Appears (e.g. x% of current window size etc.)

### ISSUE THREAD

**Issue by traverseAI** [cozyazure/leaflet-overlay#10]
Should not follow the original pixel size of image.

**Comment by cozyazure** [comment]
But how should we determine the value of x?

**Comment by traverseAI** [comment]
Till it "looks good". I think 1/6 or a bit more of the map window seems
about right.

**Comment by cozyazure** [comment]
So, regardless of zoom level, and regardless of the dimension, we always shrink it to 1/6 of the window of the browser?

**Comment by traverseAI** [comment]
Yes. 1/6 is assuming it is a vertical image but it may not be. So we can
just play around with some numbers.

The intent is that of the "overlay". If you think about powerpoint for
example, you load a 6000 pixel image, you have to zoom all the way out and
resize the image.

Thet

**Comment by cozyazure** [comment]
Okay I will think of a more intuitive way of doing this.
